packages/components/src/templates/next/components/internal/Navbar/MobileNavMenu/MobileNavbarMenu.tsx

```diff
@@ -42,6 +44,19 @@ export const MobileNavMenu = forwardRef<HTMLDivElement, MobileNavMenuProps>(
       >
         <FocusScope contain restoreFocus>
           <div className="absolute inset-0 overflow-auto border-t border-t-base-divider-subtle bg-white">
+            {callToAction && (
+              <div className="border-y border-b-base-divider-subtle bg-base-canvas-alt px-6 py-3">
+                <LinkButton
+                  href={callToAction.referenceLinkHref}
+                  isExternal={callToAction.isExternal}
+                  className="h-fit w-full justify-center"
+                  isWithFocusVisibleHighlight
+                  LinkComponent={LinkComponent}
+                >
+                  {callToAction.label}
+                </LinkButton>
```

Comment:
[issue] because of our types, we can have a link button shown without a `referenceLinkHref` - should we tighten this up?